python/src/mas/cli/install/argParser.py

```diff
@@ -143,6 +143,11 @@ def isValidFile(parser, arg) -> str:
     action="store_const",
     const="false"
 )
+masAdvancedArgGroup.add_argument(
+    "--manual-certificates",
+    required=False,
+    help=""
```

Comment:
Please add a help message here (and update the help message that is included in docs/commands/install.md)

Reply:
message added in both files